service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/ManageHearingsController.java

```diff
@@ -156,9 +159,34 @@ public CallbackResponse validateHearingDatesMidEvent(@RequestBody CallbackReques
         CaseDetails caseDetails = callbackRequest.getCaseDetails();
         CaseData caseData = getCaseData(caseDetails);
 
-        List<String> errors = validateGroupService.validateGroup(caseData, HearingDatesGroup.class);
+        List<String> errors;
+
+        if (featureToggleService.isAddHearingsInPastEnabled() && (caseData.getHearingOption() == NEW_HEARING
```

Comment:
`NEW_HEARING.equals(caseData.getHearingOptions()` for extra safety against NPE

Reply:
I think for enums == is preferred and doesn't throw null pointer - used in other places in this class

see here also for reference: https://stackoverflow.com/questions/1750435/comparing-java-enum-members-or-equals